src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -9311,7 +9311,7 @@ void Player_Init(Actor* thisx, PlayState* play2) {
     Player_SetEquipmentData(play, this);
     this->prevBoots = this->currentBoots;
     Player_InitCommon(this, play, gPlayerSkelHeaders[((void)0, gSaveContext.linkAge)]);
-    this->giObjectSegment = (void*)(((u32)ZeldaArena_MallocDebug(0x3008, "../z_player.c", 17175) + 8) & ~0xF);
+    this->giObjectSegment = (void*)(((uintptr_t)ZeldaArena_MallocDebug(0x3008, "../z_player.c", 17175) + 8) & ~0xF);
```

Comment:
Should this be an ALIGN?

Reply:
It can't really be an ALIGN since it's `+ 8) & ~0xF`, unless you mean making a macro specifically for this

Reply:
Turning `+ 8) & ~0xF` into `-7 + 0xF) & ~0xF` should make it usable with the `ALIGN16` macro, is that something worth considering?

Reply:
There's a similar one in z_room:
https://github.com/zeldaret/oot/blob/9647f50451f26fa8b96653bf792f86c6148f6d12/src/code/z_room.c#L545

Reply:
(Or was that in MM's version?)

Reply:
I don't think doing - 7 and using an ALIGN is appropriate since it's confusing and hides the fact that this relies on the existing 8 bytes alignment from ZeldaArena_MallocDebug (which is why `+ 8) & ~0xF` is enough).

I would argue the same for that z_room case (funnily enough I originally did that ALIGN but I think we should consider removing it)